### DISCUSSION TITLE
alignment of the taskbar: can't change to "center"

### DISCUSSION THREAD

**Discussion by xpaco** [valinet/ExplorerPatcher#494] [Ideas]
Hi! Thank you for your great job!

Please, adjust ExplorerPatcher for that we can center alignment in task bar 🙏

**Comment by xpaco** [comment]
done with https://github.com/valinet/ExplorerPatcher/issues/33
Thank you!

**Comment by buraCharles** [comment]
hello;
ExplorerPatcher + TaskBarX  + option "Start Menu"  It works but complicated to manage.
At least one of the 2 options in Explorerpatcher should at least be installed.
These 3 options exist in StartAllBack.
Thank you Mr. Valinet

**Reply by valinet** [reply]
You’re invited to contribute. I am not aiming for feature parity with StartAllBack anyway. I hardly understand the role of such remarks, tbh. Thanks.

**Reply by buraCharles** [reply]
This is not a criticism I made.
I only tested these 3 options together with earlystart in service
And I gave up.
it's too complicated

my matteral language is not English
Translation is sometimes bad.
